### DISCUSSION TITLE
Não abre o Blockchain [BUG]

### DISCUSSION THREAD

**Discussion by Henrique0002** [Chia-Network/chia-blockchain#3254] [Support]
**Describe the bug**
Fica em um loop infinito "conectando à carteira", ja tentei várias vezes fechar e abrir denovo mas não funciona.

**To Reproduce**

Não tem ...
**Expected behavior**

**Screenshots**
I![image](https://user-images.githubusercontent.com/83238083/116122127-c167a600-a697-11eb-98b8-6cef178eca22.png)

**Desktop (please complete the following information):**

- OS: Windows 10 Home 
- OS Version/Flavor: 10.0.19042
- CPU: Intel I5 7200U 

**Additional context**
Antes eu conseguia conectar mas do nada parou, não atualizei nem nada, apenas parou

**Comment by ZEROQUATRO** [comment]
mesma coisa comigo.......e nao faço ideia de onde encontrar suporte pra resolver, que joia...

**Comment by Bortolotiuser** [comment]
aqui mesma coisa

**Comment by renatotbz** [comment]
Aqui tava normal, comprei 2 HD de 3 TB cada... Desliguei, instalei e liguei... Nessa que liguei pra configurar... Puff BUGOU ! Já atualizei e nada.
![ERRO](https://user-images.githubusercontent.com/83735854/117222502-f2f01800-ade1-11eb-8ffb-cb2603544cd9.png)

**Comment by Bortolotiuser** [comment]
Rapaziada.. formatei o ssd e coloquei o win 10 PRO.. voltou a conectar normalmente.. ele não estava reconhecendo os plots também e voltou a reconhecer.

**Reply by renatotbz** [reply]
Que parada sem sentido... Formatar o windows pra voltar a funcionar o software, deve ser algum erro de DLL entre o software deles o Windows então... Aqui tá com o PRO já... Fogo é formatar só pra voltar a funcionar... Tira uma dúvida, depois que instalar e por a carteira ele identifica os Plots sozinho ? os que já estão montados no HD Final ??
![WINDOWS](https://user-images.githubusercontent.com/83735854/117226836-e375cc80-adeb-11eb-84d9-4d8504bc726c.png)

**Reply by Bortolotiuser** [reply]
Ele reconhece assim que add o diretório com os plots em "lotes"

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
